codespell_lib/data/dictionary.txt

```diff
@@ -11878,10 +12064,12 @@ completess->completes, completeness,
 completetion->completion
 completetly->completely
 completiom->completion
-completition->completion
+completition->completion, competition,
+completitions->completions, competitions,
 completley->completely
 completly->completely
 completness->completeness
+complets->completes
```

Comment:
I guess `couplets` and `compels` are too different, aren't they?

Reply:
I couldn't find these typos.